actors/market/src/state.rs

```diff
@@ -115,265 +153,417 @@ impl State {
             + &self.total_client_storage_fee
     }
 
-    pub(super) fn mutator<'bs, BS: Blockstore>(
-        &mut self,
-        store: &'bs BS,
-    ) -> MarketStateMutation<'bs, '_, BS> {
-        MarketStateMutation::new(self, store)
-    }
-}
+    pub(super) fn deal_states_get<BS>(
```

Comment:
Please remove the `(super)` from everything here. This is not a standard or helpful pattern in the actors.

Reply:
Rename to `find_deal_state`. As general pattern, use `find` if it returns an `Option`, or `get` if it will return an error result when the key is absent.

Reply:
As noted above, remove `anyhow` from all return types here. Use the `with_context_code()` trait method to convert to `Result<_, ActorError>`.

Reply:
There are some similar methods landing to `next` branch in https://github.com/filecoin-project/builtin-actors/pull/850/files. Could you replicate those here to reduce the eventual diff?

Reply:
Thanks @anorth for detailed comments, most of them got fixed.

**Like to align on two points.**

1, Regarding comments "There are some similar methods landing to next branch", right now base is master, can I re-base it to branch "next" ? does it make merger effort easy ?

2, Can I have quick review on naming for function on "state.rs" module.

Reply:
1. Thanks, but no `master` is the right base here
2. Don't bring in net new methods, like the new exported ones. But you can copy bits from state.rs or helpers if useful
3. Will review